comparisons/matrix_RE.txt

```diff
@@ -214,8 +221,13 @@ RunJetHT2018DecalGPUwf136p888512 136.888512_RunJetHT2018D+*ECALOnlyGPU*/step3.ro
 RunJetHT2018DhcalCPUwf136p888521 136.888521_RunJetHT2018D+*HCALOnlyCPU*/step3.root reRECO
 RunJetHT2018DhcalGPUwf136p888522 136.888522_RunJetHT2018D+*HCALOnlyGPU*/step3.root reRECO
 RunParkingBPH2018Bwf136p898 136.898_RunParkingBPH2018B+*/step3.root reRECO
+MuonEGdata106Xul18nanowf2500p332 2500.332_data106Xul18v2+MuonEG2018MINIAOD*/step2.root NANO
+
 #
 # Run3 data
+MuonEGdata124XnanoV10wf2500p510 2500.51_data124Xrun3_v10+MuonEG2022MINIAOD*/step2.root NANO
+MuonEGdata124Xnanowf2500p511 2500.511_data124Xrun3+MuonEG2022MINIAOD*/step2.root NANO
```

Comment:
```suggestion
MuonEG2022D124XnanoV10wf2500p51 2500.51_data124Xrun3_v10+MuonEG2022MINIAOD*/step2.root NANO
MuonEG2022D124Xnanowf2500p511 2500.511_data124Xrun3+MuonEG2022MINIAOD*/step2.root NANO
```
Please keep the origin information in the short names. Simply 124X is not enough.

Reply:
with only one version of mini supported in input to nano in 124X, I think there is no need for a longer useless name.
please suggest a replacement though